### PR TITLE
Fix the diagram (remove wrong comment, it was breaking the rendering).

### PR DESCRIPTION
**Pull request by t7ko-seagate**
Simply removing the comment -- diagram itself does not change.

### CODE REVIEW

**Review by evgeniy-brazhnikov — APPROVED** [review]
Any of "code owners" should approve too

**Review by sachitanands — APPROVED** [review]

**Review by vimalraghwani20 — APPROVED** [review]

**Comment by vimalraghwani20** [comment]
merged in to main

**Pull request merged**
